main.tf

```diff
@@ -41,3 +41,34 @@ resource "ibm_resource_tag" "scc_wp_access_tag" {
   tags        = var.access_tags
   tag_type    = "access"
 }
+
+##############################################################################
+# Context Based Restrictions
+##############################################################################
+module "cbr_rule" {
+  count            = length(var.cbr_rules) > 0 ? length(var.cbr_rules) : 0
+  source           = "terraform-ibm-modules/cbr/ibm//modules/cbr-rule-module"
+  version          = "1.24.0"
+  rule_description = var.cbr_rules[count.index].description
+  enforcement_mode = var.cbr_rules[count.index].enforcement_mode
+  rule_contexts    = var.cbr_rules[count.index].rule_contexts
+  resources = [{
```

Comment:
shall we add support for tags as well?
similar as we have for cos buckets https://github.com/terraform-ibm-modules/terraform-ibm-cos/blob/b9cc0f04153b289e582f80175d9bc93d648189e3/main.tf#L376